zproject/test_extra_settings.py

```diff
@@ -186,10 +185,6 @@ def set_loglevel(logger_name: str, level: str) -> None:
 SOCIAL_AUTH_APPLE_TEAM = "TEAMSTRING"
 SOCIAL_AUTH_APPLE_SECRET = get_from_file_if_exists("zerver/tests/fixtures/apple/private_key.pem")
 
-EXAMPLE_JWK = get_from_file_if_exists("zerver/tests/fixtures/example_jwk")
-APPLE_ID_TOKEN_GENERATION_KEY = get_from_file_if_exists(
-    "zerver/tests/fixtures/apple/token_gen_private_key"
-)
```

Comment:
The commit message should comment on why we can delete these.

Reply:
Oh, these hunks just belong in the next commit.

Reply:
Oh, that's a mistake. Fixed.